### PR TITLE
💅✨  UI polish

### PR DESCRIPTION
**Pull request by henryboldi**
Covers a few issues mentioned by @VladUXUI in #1078:

Asset Card

- [x] "Send button should be clickable"
- [x] "The hover effect doesn't change the Send icon color, thus making it hard to see"

Swap Page

- [x] "Margin top seems to be 17px, let's change this to 8px. This way you don't have to scroll to see the full button."

And adds a notice that ETH swaps are not yet available.

warning1

### CODE REVIEW

File: ui/components/Wallet/WalletAssetListItem.tsx [hunk 1]
```diff
@@ -58,7 +60,20 @@ export default function WalletAssetListItem(props: Props): ReactElement {
             </div>
           </div>
           <div className="right">
-            <span className="icon_send_asset" />
+            {!contractAddress && (
+              <Link
```
Comment (greg-nagy) [inline ui/components/Wallet/WalletAssetListItem.tsx:64]: This change gives the send icon 0px height and renders it invisible.

File: ui/components/Wallet/WalletAssetListItem.tsx [hunk 2]
```diff
@@ -58,7 +60,20 @@ export default function WalletAssetListItem(props: Props): ReactElement {
...
+                to={{
+                  pathname: "/send",
+                  state: {
+                    symbol: assetAmount.asset.symbol,
+                    contractAddress,
+                  },
+                }}
+                className="icon_send_asset"
```
Comment (greg-nagy) [inline ui/components/Wallet/WalletAssetListItem.tsx:72]: I think this icon and the span are duplicates.

**Review by greg-nagy — CHANGES_REQUESTED** [review]
This PR is a beauty :)

But the send button dissapers on my machine (osx, chrome) becuase of the 0px height of the icon. :(

File: ui/components/Wallet/WalletAssetListItem.tsx (hunk 2 above)
Comment (henryboldi) [reply ui/components/Wallet/WalletAssetListItem.tsx:72]: Crap I think I missed some changes in my commit on this. Ty for catching it

File: ui/components/Wallet/WalletAssetListItem.tsx (hunk 1 above)
Comment (henryboldi) [reply ui/components/Wallet/WalletAssetListItem.tsx:64]: Should be fixed now :)

**Review by greg-nagy — APPROVED** [review]
I think this PR is good improvmenet so let's 🚀

Our current implementation diverges from the design in the usage of send and swap icons but that's good for a follow up pr

design: 
CleanShot 2022-03-09 at 09 18 33

current implementation:
CleanShot 2022-03-09 at 09 14 00

**Pull request merged**